rvic/parameters.py

```diff
@@ -576,6 +576,11 @@ def gen_uh_run(uh_box, fdr_data, fdr_vatts, dom_data, outlet, config_dict,
     outlet.cell_id_source = dom_data['cell_ids'][y, x]
     outlet.x_source = x
     outlet.y_source = y
+
+    # Make sure the inds are all greater than zero GH79
```

Comment:
Github #79.  This is a common shorthand referencing the motivating issue for a particular change.  I'm happy to elaborate in the comment but I don't think it is necessary.